MetaMask/metamask-extension
# Changing cancel button labels in send screen when in editing stage

Fixes: #11929

Explanation:  White sending in metamask when editing transaction cancel buttons take user back to confirmation screen and not metamask home. This is confusing for users. As recommended in reported issue the PR change the label of edit screen when in edit stage. And Reject button at bottom ot edit send screen should take user to home page.

Manual testing steps:  
- go to send and enter a destination address - label on top right should be "Cancel" and bottom-left should also be "Cancel"
- click next
- click edit - label on top right should be "Cancel Edit" and bottom-left should be "Reject"
- Clicking top right "Cancel Edit" will take user back on previous send screen
- Clicking on bottom left "Reject" will take user back to home page

Thus you can see that label in Draft stage are still "Cancel" but they are updated in editing stage.

Send screen when drafting transaction:
<img width="425" alt="Screenshot 2021-09-10 at 10 45 07 AM" src="https://user-images.githubusercontent.com/2182307/132803291-52be24f0-b567-4196-8aeb-f6211429887c.png">

Send screen when editing transaction:
<img width="444" alt="Screenshot 2021-09-10 at 10 43 58 AM" src="https://user-images.githubusercontent.com/2182307/132803307-d4ed73e3-9093-486b-9f94-f8e1d94982ad.png">
